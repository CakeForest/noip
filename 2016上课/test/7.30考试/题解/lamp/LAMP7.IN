400
0 200 200 10000
1 4 0 9296
1 6 0 9209
1 9 0 3081
1 8 0 694
1 3 0 5144
1 2 0 452
1 3 0 4695
1 2 0 2287
1 7 0 8931
1 7 0 5635
1 6 0 9169
1 5 0 2587
1 9 0 7445
1 4 0 4789
1 2 0 7089
1 8 0 1136
1 6 0 7198
1 2 0 7583
1 1 0 7788
1 1 0 3817
1 1 0 1696
1 3 0 9025
1 7 0 720
1 6 0 6794
1 5 0 3752
1 2 0 7794
1 9 0 1509
1 4 0 8317
1 0 0 3820
1 9 0 9053
1 8 0 8349
1 1 0 9701
1 6 0 1357
1 6 0 3859
1 9 0 5512
1 6 0 1742
1 4 0 6191
1 6 0 6379
1 5 0 7108
1 4 0 7271
1 0 0 4379
1 8 0 4688
1 2 0 1529
1 6 0 976
1 3 0 495
1 7 0 7472
1 5 0 5288
1 5 0 1712
1 2 0 4050
1 2 0 5430
1 7 0 8863
1 3 0 5839
1 5 0 1866
1 9 0 2990
1 8 0 536
1 3 0 6134
1 9 0 3598
1 7 0 3068
1 2 0 4179
1 4 0 7948
1 6 0 4948
1 8 0 3150
1 7 0 1469
1 6 0 5298
1 1 0 2810
1 9 0 6107
1 2 0 8005
1 9 0 5280
1 8 0 364
1 9 0 5119
1 0 0 8248
1 9 0 8813
1 3 0 7939
1 3 0 5228
1 2 0 5937
1 7 0 8368
1 2 0 640
1 3 0 4448
1 9 0 2545
1 5 0 2087
1 9 0 4363
1 6 0 9593
1 1 0 6610
1 7 0 3067
1 1 0 6242
1 5 0 8872
1 6 0 5402
1 3 0 7488
1 4 0 8594
1 5 0 3172
1 7 0 1849
1 0 0 4231
1 3 0 3341
1 8 0 657
1 4 0 2646
1 3 0 7699
1 1 0 9672
1 9 0 9387
1 4 0 7573
1 1 0 8883
1 0 0 573
1 0 0 9199
1 8 0 2037
1 8 0 8405
1 5 0 6658
1 9 0 4342
1 6 0 5216
1 2 0 2700
1 3 0 3216
1 7 0 6810
1 5 0 3301
1 5 0 7955
1 5 0 6879
1 0 0 4679
1 3 0 2490
1 5 0 8580
1 5 0 1667
1 5 0 22
1 3 0 7187
1 4 0 1285
1 1 0 4158
1 9 0 2059
1 9 0 5404
1 9 0 8222
1 9 0 4079
1 7 0 9976
1 8 0 9808
1 6 0 5675
1 6 0 6464
1 7 0 2531
1 7 0 6077
1 7 0 5459
1 4 0 4690
1 1 0 8968
1 2 0 8979
1 1 0 5696
1 6 0 5769
1 3 0 1195
1 0 0 9187
1 0 0 6907
1 2 0 9788
1 7 0 5550
1 1 0 7936
1 6 0 1224
1 6 0 2232
1 3 0 6876
1 5 0 9361
1 3 0 4542
1 7 0 4330
1 9 0 9323
1 8 0 4407
1 6 0 9357
1 7 0 2230
1 3 0 7757
1 8 0 8235
1 8 0 9886
1 4 0 9612
1 7 0 367
1 0 0 1028
1 7 0 3990
1 0 0 3808
1 0 0 4485
1 8 0 7223
1 9 0 6839
1 5 0 262
1 7 0 863
1 3 0 6690
1 2 0 238
1 0 0 2966
1 6 0 406
1 7 0 4890
1 6 0 4371
1 5 0 2983
1 0 0 1845
1 3 0 7129
1 3 0 9190
1 1 0 362
1 0 0 5964
1 4 0 7999
1 3 0 3158
1 9 0 277
1 3 0 2368
1 4 0 4062
1 7 0 1515
1 3 0 3488
1 4 0 6189
1 0 0 8153
1 4 0 1221
1 8 0 5255
1 1 0 9075
1 7 0 9806
1 6 0 2446
1 0 0 6894
1 3 0 7008
1 0 0 4797
1 9 0 7447
1 4 0 2437
1 1 0 4219
1 5 0 5819
1 3 0 811
1 7 0 3826
1 1 0 7269
1 7 0 7980
1 3 0 1976
1 9 0 6383
1 7 0 1086
1 2 0 8176
1 4 0 8847
1 2 0 7429
1 2 0 4100
1 6 0 8613
1 6 0 2225
1 1 0 4026
1 3 0 6665
1 8 0 5108
1 0 0 1501
1 4 0 1635
1 1 0 1937
1 3 0 479
1 2 0 5085
1 9 0 5758
1 2 0 605
1 7 0 6211
1 1 0 5146
1 6 0 2132
1 4 0 7698
1 9 0 4618
1 1 0 4922
1 4 0 1555
1 7 0 2439
1 5 0 4074
1 5 0 1891
1 2 0 1905
1 1 0 2388
1 3 0 648
1 4 0 9259
1 0 0 1329
1 7 0 5636
1 5 0 5625
1 7 0 1800
1 9 0 9981
1 3 0 1174
1 1 0 3243
1 1 0 8785
1 9 0 6266
1 2 0 8347
1 5 0 4104
1 2 0 5562
1 2 0 8935
1 9 0 2826
1 6 0 8621
1 7 0 2953
1 5 0 5402
1 4 0 3221
1 0 0 8870
1 5 0 3957
1 4 0 2951
1 0 0 2915
1 4 0 999
1 5 0 3966
1 1 0 352
1 1 0 2488
1 9 0 3557
1 4 0 6844
1 7 0 996
1 6 0 2396
1 7 0 4353
1 9 0 3088
1 5 0 9263
1 8 0 8145
1 6 0 866
1 9 0 8514
1 0 0 962
1 6 0 1868
1 2 0 2275
1 0 0 2457
1 9 0 9478
1 1 0 2989
1 3 0 7907
1 7 0 4353
1 1 0 1066
1 0 0 4121
1 4 0 749
1 8 0 4508
1 1 0 7
1 2 0 5428
1 2 0 5088
1 2 0 5573
1 8 0 5399
1 1 0 2390
1 4 0 9291
1 8 0 6324
1 2 0 8837
1 2 0 3874
1 6 0 1582
1 6 0 8357
1 0 0 5008
1 2 0 5947
1 1 0 6335
1 3 0 6596
1 9 0 1596
1 2 0 1277
1 0 0 2597
1 0 0 2800
1 8 0 3816
1 0 0 5269
1 8 0 2916
1 6 0 8775
1 2 0 2453
1 3 0 1542
1 2 0 5863
1 4 0 5694
1 7 0 7886
1 5 0 2777
1 3 0 5503
1 5 0 4539
1 0 0 7882
1 6 0 9918
1 4 0 5820
1 9 0 3877
1 2 0 5507
1 6 0 3823
1 6 0 1543
1 2 0 7712
1 1 0 8598
1 6 0 9951
1 9 0 3190
1 6 0 6519
1 1 0 1046
1 2 0 5297
1 0 0 2580
1 4 0 5014
1 6 0 8696
1 4 0 6721
1 4 0 2913
1 6 0 8923
1 8 0 3664
1 0 0 3150
1 9 0 1768
1 9 0 4624
1 8 0 7204
1 7 0 4240
1 7 0 7963
1 7 0 7464
1 9 0 4386
1 3 0 5819
1 7 0 6177
1 2 0 4837
1 3 0 3644
1 6 0 4
1 6 0 1148
1 9 0 9062
1 5 0 5036
1 2 0 7132
1 6 0 9277
1 8 0 8891
1 0 0 4952
1 4 0 2917
1 0 0 7242
1 7 0 9688
1 2 0 8552
1 2 0 6162
1 8 0 331
1 1 0 5259
1 8 0 3989
1 0 0 2201
1 3 0 4226
1 4 0 2398
1 8 0 41
1 6 0 4927
1 9 0 9756
1 3 0 7179
1 9 0 1279
1 9 0 641
1 9 0 7678
1 4 0 2562
1 6 0 9653
1 1 0 8771
1 6 0 5661
1 5 0 2595
1 9 0 1843
1 2 0 9855
1 5 0 6373
1 7 0 8232
1 0 0 121
1 6 0 1661
1 5 0 1189
1 2 0 1380
1 3 0 5512
1 5 0 5815
1 8 0 6911
1 3 0 9685
1 8 0 9106
1 0 0 1260
1 4 0 3739
1 5 0 5118
1 8 0 7404
1 5 0 848
1 2 0 7610
