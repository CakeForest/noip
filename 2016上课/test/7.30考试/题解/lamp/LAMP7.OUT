130 799913
1 2 16 19 20 21 23 27 29 30 33 42 50 51 53 57 60 61 66 68 72 74 76 84 86 89 90 93 97 98 100 101 102 103 115 120 122 135 136 137 140 141 142 144 147 148 155 158 160 162 163 168 170 175 176 177 179 180 188 191 194 195 196 199 203 208 209 210 211 214 215 217 221 225 227 229 235 237 238 244 245 247 249 250 256 259 263 274 277 279 280 283 287 288 289 291 292 294 295 298 299 300 301 304 305 307 312 318 322 325 326 331 332 339 347 349 355 358 360 362 363 365 367 373 379 383 386 393 395 400